Stacks on TFE GA (#2018)
hashicorp/web-unified-docs: content/terraform-docs-common/docs/cloud-docs/vcs/github.mdx

## Finished

> At this point, GitHub access for HCP Terraform is fully configured, and you can create Terraform workspaces or Stacks based on your organization's shared GitHub repositories.

Author: ```suggestion
You can now create workspaces or Stacks based on your organization's shared GitHub repositories.
```
